backend/web/src/frontend/src/views/header/component/TagNavigationBody.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-card :flat="true">
+    <v-list :key="category.id" class="grow" v-for="category in categories">
+      <CategoryGroup :category="category" />
+    </v-list>
+  </v-card>
+</template>
+
+<script>
+import { FETCH_CATEGORIES } from '@/store/share/actionTypes.js';
+import { mapActions, mapGetters } from 'vuex';
+import { SEARCHED_CATEGORIES, TOTAL_CATEGORIES } from '@/store/share/getterTypes.js';
+import CategoryGroup from '@/views/header/component/CategoryGroup.vue';
+
+export default {
+  name: 'TagNavigationBody',
+  components: {
+    CategoryGroup,
+  },
+  props: {
+    searchKeyword: {
+      type: String,
+    },
+  },
+  computed: {
+    ...mapGetters([TOTAL_CATEGORIES, SEARCHED_CATEGORIES]),
+    categories() {
+      if (this.searchKeyword === '') {
+        return this[TOTAL_CATEGORIES];
+      }
+      return this[SEARCHED_CATEGORIES](this.searchKeyword);
+    },
+  },
+  created() {
+    this[FETCH_CATEGORIES]();
+  },
+  methods: {
+    ...mapActions([FETCH_CATEGORIES]),
+  },
+};
+</script>
```

Comment:
```suggestion
</script>

<style>
.v-list {
  padding: 0 !important;
}
</style>
```
태그 목록이 두꺼워서 padding을 제거해줘도 좋을 것 같아요